xbmc/filesystem/PluginDirectory.cpp

```diff
@@ -238,6 +238,12 @@ void CPluginDirectory::AddSortMethod(int handle, SORT_METHOD sortMethod, const s
         dir->m_listItems->AddSortMethod(SortBySortTitle, 556, LABEL_MASKS(labelMask, "%M", labelMask, "%M"), CServiceBroker::GetSettingsComponent()->GetSettings()->GetBool(CSettings::SETTING_FILELISTS_IGNORETHEWHENSORTING) ? SortAttributeIgnoreArticle : SortAttributeNone);
         break;
       }
+	case SORT_METHOD_VIDEO_ORIGINAL_TITLE:
+	case SORT_METHOD_VIDEO_ORIGINAL_TITLE_IGNORE_THE:
+	  {
+        dir->m_listItems->AddSortMethod(SortByOriginalTitle, 556, LABEL_MASKS(labelMask, "%M", labelMask, "%M"), CServiceBroker::GetSettingsComponent()->GetSettings()->GetBool(CSettings::SETTING_FILELISTS_IGNORETHEWHENSORTING) ? SortAttributeIgnoreArticle : SortAttributeNone);
```

Comment:
We are working towards making all code to the same standard so new lines of code needs to be to our current style guides rather than copy the style around it. This looks way too long. Run `clang` on the changed/added lines of code.

Reply:
I dont understand. I followed existing format of a file which I changed (SORT_METHOD_VIDEO_SORT_TITLE_IGNORE_THE for reference). This is minor change, not refactor bigger chunks of application like filesystem part. I dont have such intentions..

Reply:
Just wait for the jenkins build to finish and it will automatically leave a comment here with the formatting changes necessary. Then you can apply those to your commits.